wiki/Getting-Started.md

```diff
@@ -1,18 +1,23 @@
 # Interested in seeing examples of Workflow in action?
+
 Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
 
 # CocoaPods with Storyboards
 
 ## Adding the dependency
+
 ```ruby
 pod 'DynamicWorkflow/UIKit'
 ```
+
 Add the above line to your Podfile.
 
-For more installation instructions, see the wiki on [installation](https://github.com/wwt/Workflow/wiki/Installation).
+For information on importing and more installation instructions, check out our [Installation](https://github.com/wwt/Workflow/wiki/Installation) page.
```

Comment:
Specifically interested in feedback on this verbiage. I feel like it should probably directly call out that SPM details are here